pkg/webhook/imagepulljob/validating/imagepulljob_create_update_handler.go

```diff
@@ -97,10 +96,17 @@ func validate(obj *appsv1alpha1.ImagePullJob) error {
 	if _, err := daemonutil.NormalizeImageRef(obj.Spec.Image); err != nil {
 		return fmt.Errorf("invalid image %s: %v", obj.Spec.Image, err)
 	}
-
+	if obj.Spec.PullPolicy == nil {
+		obj.Spec.PullPolicy = &appsv1alpha1.PullPolicy{}
+	}
+	if obj.Spec.PullPolicy.TimeoutSeconds == nil {
+		obj.Spec.PullPolicy.TimeoutSeconds = utilpointer.Int32Ptr(600)
+	}
 	switch obj.Spec.CompletionPolicy.Type {
 	case appsv1alpha1.Always:
-
+		if obj.Spec.CompletionPolicy.ActiveDeadlineSeconds != nil && int64(*obj.Spec.PullPolicy.TimeoutSeconds) > *obj.Spec.CompletionPolicy.ActiveDeadlineSeconds {
+			return fmt.Errorf("activeDeadlineSeconds must be greater than pullPolicy.timeoutSeconds(default 600)")
```

Comment:
"completionPolicy.activeDeadlineSeconds must be greater than pullPolicy.timeoutSeconds(default 600)"